docs/connector-development/tutorials/building-a-python-source.md

```diff
@@ -213,7 +213,7 @@ The template fills in most of the information for the readme for you. Unless the
 
 ### Step 11: Add the connector to the API/UI
 
-Open the following file: `airbyte-config/init/src/main/resources/seed/source_definitions.yaml`. You'll find a list of all the connectors that Airbyte displays in the UI. Pattern match to add your own connector. Make sure to generate a new _unique_ UUIDv4 for the `sourceDefinitionId` field. You can get one [here](https://www.uuidgenerator.net/).
+Open the following file: `airbyte-config/init/src/main/resources/seed/source_definitions.yaml`. You'll find a list of all the connectors that Airbyte displays in the UI. Pattern match to add your own connector. Make sure to generate a new _unique_ UUIDv4 for the `sourceDefinitionId` field. You can get one [here](https://www.uuidgenerator.net/). Note that modifications to `source_definitions.yaml` will only be picked-up the first time you start airbyte, or when you upgrade airbye, or if you entirely wipe our your instance of airbyte and start from scratch. 
```

Comment:
Capitalization and spelling nit: Note that modifications to `source_definitions.yaml` will only be picked-up the first time you start Airbyte, or when you upgrade Airbyte, or if you entirely wipe our your instance of Airbyte and start from scratch.